hoot-core/src/main/cpp/hoot/core/io/OsmApiChangesetElement.cpp

```diff
@@ -43,13 +45,38 @@ XmlElement::XmlElement(const XmlObject& object, ElementIdToIdMap* idMap)
 {
 }
 
+XmlElement::XmlElement(const XmlElement& element)
+  : _type(element._type),
+    _id(element._id),
+    _version(element._version),
+    _object(element._object),
+    _tags(element._tags),
+    _idMap(element._idMap),
+    _status(element._status)
+{
+}
+
```

Comment:
All of the work in this file is to allow for the copies of these element based objects to keep their tags so that tags can be updated if the original element changeset fails.